businesscentral/image.yaml

```diff
@@ -1,13 +1,13 @@
 name: "jboss-bpmsuite-7/bpmsuite70-businesscentral"
 description: "Red Hat JBoss BPM Suite 7.0 Business Central container image"
-version: "7.0.0"
+version: "7.0.0.CAP02"
 from: "jboss-eap-7/eap70:latest"
 user: 185
 envs:
     - name: "JBOSS_PRODUCT"
-      value: "bpmsuite"
-    - name: "JBOSS_BPMSUITE_VERSION"
-      value: "7.0.0"
+      value: "bpmsuite-businesscentral"
+    - name: "JBOSS_BPMSUITE_BUSINESSCENTRAL_VERSION"
+      value: "7.0.0.CAP02"
```

Comment:
Changing this env I just intended to express that the version of the inner artifact used was that one. The complete product version could still be a different one. In any case, I'll change all back to 7.0.0